src/group_theory/free_group.lean

```diff
@@ -820,6 +829,15 @@ by rintros ⟨L⟩; exact reduce.self
 lemma to_word.inj : ∀(x y : free_group α), to_word x = to_word y → x = y :=
 by rintros ⟨L₁⟩ ⟨L₂⟩; exact reduce.exact
 
+theorem to_word.le_length (x : free_group α) (h : free_group.mk L₁ = x) : x.to_word.length ≤ L₁.length := 
```

Comment:
```suggestion
theorem to_word.le_length (x : free_group α) (h : free_group.mk L₁ = x) :
  x.to_word.length ≤ L₁.length := 
```

Reply:
Usually, we don't use assumptions like `free_group.mk L = x`. Instead, you can formulate your lemma like `(free_group.mk L).to_word.length ≤ L.length`